terraform/Makefile

```diff
@@ -16,6 +16,9 @@ plan: validate
 apply: validate
 	terraform apply
 
+grafana: apply
+	python ../scripts/update_grafana.py
```

Comment:
Use `$(azul_home)` instead of a relative path. A relative path with `..` in it hampers relocatability of this directory.